emulator/iSkipper/iClickerEmulator.cpp

```diff
@@ -198,24 +198,30 @@ bool iClickerEmulator::submitAnswer(uint8_t id[ICLICKER_ID_LEN], iClickerAnswer
 }
 
 void iClickerEmulator::acknowledgeAnswer(iClickerAnswerPacket* packet, bool accept) {
-    uint8_t ack_payload[ACK_SIZE];
+    // ACKs are either 4 (accept) or 5 (reject) bytes
+    uint8_t ack_payload[5];
     encodeId(packet->id, ack_payload);
 
-    ack_payload[2] = ~ack_payload[2];
+    configureRadio(CHANNEL_RECV, ACK_SEND_SYNC_ADDR);
```

Comment:
One thing I am a bit worried about is `configureRadio(CHANNEL_RECV, ACK_SEND_SYNC_ADDR)` calls `  _radio.setChannelType(type)` which calls `RFM69::setPayloadLength()` which sets the payload length on the radio's register. Currently, the payload length is being set to 7, but only 4 or 5 bytes is actually being transmitted. I'm not sure if this is safe to do.

The macro `PAYLOAD_LENGTH_RECV` should be set to 5, and I think we should always just transmit 5 bytes. In the reject case, the last byte can just be a 0. This should not affect the receiver at all, and would fix this payload length mismatch problem.

Reply:
> Currently, the payload length is being set to 7, but only 4 or 5 bytes is actually being transmitted. I'm not sure if this is safe to do.

The payload length is only used for receiving, during transmission all of the bytes in the FIFO of the chip are sent, `send` places all the bytes in the FIFO during sleep mode and then puts the radio in send mode which flushes out all the bytes or if `RF_FIFOTHRESH_TXSTART_FIFOTHRESH_IC` is set it waits until `n+1` bytes are there. 

Search for `0x3C` in the datasheet for more details https://cdn.sparkfun.com/datasheets/Wireless/General/RFM69HCW-V1.1.pdf

Reply:
Good to know! I did not know this. 

Regardless though `PAYLOAD_LENGTH_RECV` should be set to 5 so promiscousMode works for capturing acks. There is no sense in having configureRadio, configure the radio to the incorrect payload length.

Reply:
Will do this for now but this code path will need refactoring since welcome messages and other question modes have different payload lengths. I'll do that in the next PR though.